packages/terra-slide-panel-manager/tests/jest/SlidePanelManager.test.jsx

```diff
@@ -1,68 +0,0 @@
-import React from 'react';
```

Comment:
Do we want to remove these jest tests?

Reply:
We should keep the jest tests @amichaelparker